module/Education/src/Education/Form/AddCourse.php

```diff
@@ -0,0 +1,93 @@
+<?php
+
+namespace Education\Form;
+
+use Zend\Form\Form;
+use Zend\InputFilter\InputFilter;
+use Zend\InputFilter\InputFilterProviderInterface;
+use Zend\I18n\Translator\TranslatorInterface as Translator;
+
+class AddCourse extends Form implements InputFilterProviderInterface
+{
+
+    public function __construct(Translator $translator)
+    {
+
+	$qOptions = ['q1' => $translator->translate('Quartile 1'),
+			'q2' => $translator->translate('Quartile 2'),
+			'q3' => $translator->translate('Quartile 3'),
+			'q4' => $translator->translate('Quartile 4'),
+			'interim' => $translator->translate('Interim')];
```

Comment:
The formatting here is definitely off. For coding standards:

- https://www.php-fig.org/psr/psr-1/
- https://www.php-fig.org/psr/psr-2/